rest/model/task.go

```diff
@@ -264,6 +263,16 @@ func (at *APITask) BuildFromService(t interface{}) error {
 			}
 		}
 
+		if v.TimeTaken != 0 {
+			at.TimeTaken = NewAPIDuration(v.TimeTaken)
+		} else {
+			if v.Status == evergreen.TaskStarted {
+				at.TimeTaken = NewAPIDuration(time.Duration(time.Since(v.StartTime).Nanoseconds()))
+			} else {
+				at.TimeTaken = 0
```

Comment:
We don't need to specify this, since that's the default value for an integer. (So then 269 can just be else if)